baselines/dom.generated.d.ts

```diff
@@ -8249,20 +8249,20 @@ interface Node extends EventTarget {
     readonly previousSibling: Node | null;
     textContent: string | null;
     appendChild<T extends Node>(newChild: T): T;
-    cloneNode(deep?: boolean): Node;
+    cloneNode(deep?: boolean): this;
```

Comment:
The other alternative is to just leave `cloneNode` the way it is and take the other changes.